### PR TITLE
Goblin and Orc Balance Adjustments

### PR DESCRIPTION
**Pull request by lavillastrangiato**
## About The Pull Request
Okay, let's try it like this.

- If the list of living players in-game is lower than 10, only 1 goblin will emerge from a goblin portal.
- Goblins no longer have infinite stamina.
- Orcs no longer have infinite stamina. Some orcs have had their stats buffed to compensate. I am of the opinion that orcs should be boss-tier mobs, but still be bound to the rules of the game.

## Why It's Good For The Game

Goblins really suck ass to fight against if you aren't a combat class, and sometimes even if you _are_ a combat class. They're difficult to outrun, their AI makes them bob and weave from your attacks, and they take a while to die even if you've got them on the ground and are currently bashing their head in with a rock.

I think it is much more conducive to fun roleplay times, especially on low-pop, if goblins can now get tired and be run down or run away from. They're certainly not nerfed into the ground, you'll still have to be cautious when you're fighting them if you're backed into a corner as a non-combat class, but there shouldn't be any more lowpop situations where you're stunlocked into a corner by two of the little fuckers.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by 1N7Slayer** [comment]
lgtm 👍

File: code/modules/mob/living/carbon/human/npc/goblin.dm [hunk 1]
```diff
@@ -104,7 +104,7 @@
 	name = "goblin"
 	id = "goblin"
 	species_traits = list(NO_UNDERWEAR,NOEYESPRITES)
-	inherent_traits = list(TRAIT_NOROGSTAM,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_RADIMMUNE, TRAIT_EASYDISMEMBER, TRAIT_CRITICAL_WEAKNESS, TRAIT_NASTY_EATER, TRAIT_LEECHIMMUNE, TRAIT_INHUMENCAMP)
```
Comment (BadAtThisGame302) [inline code/modules/mob/living/carbon/human/npc/goblin.dm:107]: Why remove inhumen camp?

Comment (lavillastrangiato) [reply code/modules/mob/living/carbon/human/npc/goblin.dm:107]: IDK how that happened, I'll go back and readd it.

Comment (NPC1314) [reply code/modules/mob/living/carbon/human/npc/goblin.dm:107]: need to fix before merge tbh

Comment (NPC1314) [reply code/modules/mob/living/carbon/human/npc/goblin.dm:107]: I'll sort it

**Review by NPC1314 — APPROVED** [review]

**Pull request merged**